src/components/site/tableOfContents/table-of-contents.module.scss

```diff
@@ -70,9 +70,13 @@
 }
 
 .table-of-contents {
-  $pageNavHeight: 168px;
   z-index: 14;
   padding-top: calc(#{$siteHeaderHeight} + 10px);
+  position: fixed;
+  bottom: 326px;
+  @include respond($break80, max) {
+    bottom: 330px;
+  }
```

Comment:
Is already `position: fixed` because of `md-drawer--fixed` class.  But also, I think this approach of "raising" the bottom of this el off the bottom of its parent el is maybe a bit of a hack.  Would you tell me a bit about how/why you landed on this solution?  I'm gonna make a suggestion below that I think is in the spirit of your solution, but coming at it in a maybe more conventional way?

Reply:
I wanted to be more 'forceful' with this approach as not to leave any room for misinterpretation from any browsers.